applications/soupbintcp-gateway/src/main/java/com/paritytrading/nassau/soupbintcp/gateway/Session.java

```diff
@@ -13,8 +12,6 @@
 
 class Session implements Closeable, MessageListener, SoupBinTCPServerStatusListener {
 
-    private static SoupBinTCP.LoginAccepted loginAccepted = new SoupBinTCP.LoginAccepted();
-
```

Comment:
Can't multiple `LoginRequest`s be processed a (nearly) the same time?

Reply:
No, the gateway uses only a single thread. Only one Login Request will be processed at a time. 🙂